### DISCUSSION TITLE
How to change direction or position

### DISCUSSION THREAD

**Discussion by rabol** [usernotnull/tall-toasts#27] [General]
Hi

How can I change the position and direction of the messages ?

**Comment by usernotnull** [comment]
Hey @rabol,

You can publish the views and display them as you like, see [here](https://github.com/usernotnull/tall-toasts#customizing-views).

This is a screenshot of a playground (unpublished) that I whipped up that shows you the toasts are as flexible as your skills in blade :)

In the screenshot below, for example, I flipped everything RTL and made it in Arabic.

![image](https://user-images.githubusercontent.com/15612814/140353330-c6f551d4-2e9c-4476-9535-1c1cd53ba560.png)

The toasts container, if that is your question specifically, can be edited from [this file](https://github.com/usernotnull/tall-toasts/blob/main/resources/views/livewire/toasts.blade.php), after you published the views.

**Reply by rabol** [reply]
hmm..

It was not the direction of the text, sorry 

I was more hoping for some kind of parameters :)

sometimes the 'toast' could come from left and sometimes the 'toast' could come from the right or from the top

**Reply by usernotnull** [reply]
I edited the above reply to mention the container. Changing the classes of the published view container is the way to go. You can display them right, left, center, top, bottom... Normal HTML stuff.

I did not make it as a config file because that way you will have full power to customize the view.

**Reply by rabol** [reply]
well, If I change the view, then the 'toast' will always come from 'top and then down' or always from 'left to right' or as it is now, bottom right.

**Reply by usernotnull** [reply]
![image](https://user-images.githubusercontent.com/15612814/140363605-df3d0c77-074c-42b3-963d-f908be78f44b.png)

I changed toasts.blade.php classes to this `toasts-container fixed bottom-0 z-50 p-4 space-y-3 w-80 sm:p-6 center-x`

center-x being `left:50%;transform:translate(-50%, 0);`

**Reply by rabol** [reply]
yes, and from now on, all the messages is in that position - I get that.
what I was hoping for was some kind of flexibility

I'll find some thing.

**Reply by usernotnull** [reply]
Okay, so your question is INDIVIDUAL styling of toasts.

The way to go is probably making the container fill the screen, then letting the toasts decide how to style and position themselves.
This requires a new toast method, like 'success', 'error', and user-added styles such as `center-alert`, `center-sticky`, etc. 

This won't be a simple feat esp if I'm aiming for it to be a simple library, but definitely doable. If you're ready for a PR pls go ahead, otherwise, I'll add it to my to-do list for the next version.
